src/discovery.ts

```diff
@@ -31,16 +32,22 @@ export class PeerDiscovery {
 				for (const seed of body.data) {
 					let port = defaultPort;
 					if (seed.ports) {
-						const walletApiPort = seed.ports["@arkecosystem/core-wallet-api"];
 						const apiPort = seed.ports["@arkecosystem/core-api"];
-						if (walletApiPort >= 1 && walletApiPort <= 65535) {
-							port = walletApiPort;
-						} else if (apiPort >= 1 && apiPort <= 65535) {
+						if (apiPort >= 1 && apiPort <= 65535) {
 							port = apiPort;
 						}
 					}
 
-					seeds.push({ ip: seed.ip, port });
+					seeds.push({
+						ip: seed.ip,
```

Comment:
Does this seed object contain any other properties? Otherwise we can simplify this to
```
seeds.push({...seed, port});
```

if yes then making an object and unwrapping it is still useless and we can simplify to:
```
seeds.push({
    ip: seed.ip, 
    port, 
    ports: seed.ports, 
    version: seed.version,
    height: seed.height,
    latency: seed.latency
});
```